### PR TITLE
fix build commands for native tenant manager builds

### PR DESCRIPTION
**Pull request by riprasad**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -171,13 +178,35 @@ push-tenant-manager-image:
...
 	docker push $(IMAGE_REPO)/apicurio/apicurio-registry-tenant-manager-api:$(IMAGE_TAG)
 
+.PHONY: build-tenant-manager-native-image ## Builds native docker image for tenant manager. Variables available for override [IMAGE_REPO, IMAGE_TAG]
+build-tenant-manager-native-image:
+	@echo "------------------------------------------------------------------------"
+	@echo " Building Native Image For Tenant Manager API"
+	@echo " Repository: $(IMAGE_REPO)"
+	@echo " Tag: $(IMAGE_TAG)"
+	@echo "------------------------------------------------------------------------"
+	docker build -f multitenancy/tenant-manager-api/src/main/docker/Dockerfile.native -t $(IMAGE_REPO)/apicurio/apicurio-registry-tenant-manager-api-native:$(IMAGE_TAG) ./multitenancy/tenant-manager-api/
```
Comment (riprasad) [inline Makefile:188]: We would need `apicurio/apicurio-registry-tenant-manager-api-native` repository in quay I guess. Dockerhub will create one when pushing for the first time, AFAIK quay doesn't

**Comment by riprasad** [comment]
@EricWittmann not sure what's the workflow for `mas-sr` branch, the changes must also be present there to be in sync

**Comment by famartinrh** [comment]
ok, my bad, let's merge whatever of this two PRs you prefer

File: Makefile (hunk 1 above)
Comment (famartinrh) [reply Makefile:188]: request the credentials and create the repo in quay please

**Review by EricWittmann — APPROVED** [review]

**Pull request merged**